bin/mail/2016-03-21-teacher-recruitment/generate-inactive-teachers

```diff
@@ -5,13 +5,14 @@ require 'set'
 # Inactive Code Studio teacher accounts  US only
 # - not teaching a course, haven't added students, or no students logged in
 
+# TODO(asher): The variable US_TEACHERS is mutated, change it to lower_case.
 US_TEACHERS = Set.new
 SOLR_PARAMS = {
   q: 'kind_s:user && role_s:teacher',
   fq: 'create_ip_country_s:"United States"',
   fl: 'email_s',
   rows: 10000
-}
+}.freeze
 SOLR.query(SOLR_PARAMS).each do |record|
   US_TEACHERS << record['email_s']
 end
```

Comment:
Probably, I'll follow up in another PR. For these cleanup PRs, I want to keep the changes to (e.g.) only adding `freeze` (also associated whitespace changes and related TODOs).